### DISCUSSION TITLE
How to implement onMouseOver for a day tile?

### DISCUSSION THREAD

**Discussion by fabio3878** [wojtekmaj/react-calendar#614] [Q&A]
Hello. I'm trying to do an onMouseOver of the day . 
Does anyone have any ideas?

**Comment by wojtekmaj** [comment]
You can capture is by attaching a listener to React-Calendar's parent and leveraging event bubbling, like so:

https://codesandbox.io/s/react-calendar-day-tile-mouseover-zh5osp
